### PR TITLE
Add Admonitions to Build >> Dev Content

### PR DESCRIPTION
**Pull request by salmad3**
Part of https://github.com/w3f/polkadot-wiki/issues/3133

### CODE REVIEW

File: docs/build/build-parachains.md [hunk 1]
```diff
@@ -88,9 +88,13 @@ passing messages between them. Furthermore, as bridges to other chains are conne
 to Bitcoin or Ethereum) {{ polkadot: Polkadot's :polkadot }}{{ kusama: Kusama's :kusama }} parachains
 will be able to communicate with these as well.
 
-> **Please note**: Despite the benefits of becoming a parachain, developers should be conscious of the 
-> challenges in becoming a parachain, and whether building a blockchain with an end goal of becoming a
-> parachain is a viable one for their project.
+:::note Building considerations
```
Comment (profradha) [inline docs/build/build-parachains.md]: Just the note should be fine
```suggestion
:::note 
```

File: docs/build/build-parachains.md [hunk 2]
```diff
@@ -236,11 +246,16 @@ Cumulus is still in development, but the idea is that it should be simple
 to take a Substrate chain and add the parachain code by importing the crates and adding a single line
 of code. Keep up-to-date with the latest Cumulus developments from the [Cumulus section](###cumulus).
 
-> Substrate and Cumulus provide a PDK from the abstraction of the blockchain format, but it is **not
-> necessary** that a parachain even needs to be a blockchain. For example, a parachain just needs to
-> satisfy the two constraints listed above: *state transition function* and *collator node*.
->
-> Everything else is up to the implementer of the PDK.
+
+:::info Using PDKs
```
Comment (profradha) [inline docs/build/build-parachains.md]: Note should suffice
```suggestion
:::info 
```

File: docs/build/build-wallets.md [hunk 3]
```diff
@@ -36,13 +36,21 @@ post where the Treasury Proposal was discussed.
...
-> **This third-party wallet was not funded by a Treasury Proposal and instead received a tip from the Treasury. Unlike a Treasury Proposal, tipping is a separate 
-> system that the council members individually participate in to collectively decide on the value of the tip. Learn more about tipping on the 
-> [Treasury Page](../learn/learn-treasury.md##tipping). 
+These third-party wallets have been funded by the community through either the Polkadot or Kusama 
+[Treasury](learn-treasury.md). You should use your own due diligence in researching and using them.
+The official Polkadot Support cannot provide support for issues with 
+these wallets or other non-Parity developed wallets.
+
+:::
+
```
Comment (profradha) [inline docs/build/build-wallets.md:39]: Can this note on the tip be moved above and closer to the table?

File: docs/build/build-storage.md [hunk 4]
```diff
@@ -71,9 +75,13 @@ built on the key-value store.
 The key-value data store is backed by [RocksDB](https://rocksdb.org/), and it also supports an 
 experimental [Parity database](https://github.com/paritytech/parity-db).
 
-> The database is used for components that require persistent storage: Substrate clients, Substrate light-clients
-> & off-chain workers. For more information, check out
-> Substrate Developer Hub's [Storage Page](https://docs.substrate.io/v3/advanced/storage/).
+:::note The database is used for components that require persistent storage
+
+These are components like Substrate clients, Substrate light-clients & off-chain workers.
```
Comment (profradha) [inline docs/build/build-storage.md:81]: This line is not reading well. Probably remove the content from the title and use the actual content inside the note

**Review by DrW3RK — APPROVED** [review]
Minor suggestions. Some notes benefit from titles, but for most of them, it is okay to leave them as is. Suggested removal of a few titles.

File: docs/build/build-wallets.md (hunk 3 above)
Comment (salmad3) [reply docs/build/build-wallets.md:39]: moved the warning before the table

**Pull request merged**